From 63af5224fa36abe4468c07697aaa93d0fd2ae1e8 Mon Sep 17 00:00:00 2001
From: nanake <nanake@users.noreply.github.com>
Date: Fri, 31 Mar 2023 12:05:33 +0900
Subject: [PATCH 2/5] ioctl

Signed-off-by: nanake <nanake@users.noreply.github.com>
---
 contrib/ntsc-cc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/ntsc-cc.c b/contrib/ntsc-cc.c
index 5f20a8d..cbeccb1 100644
--- a/contrib/ntsc-cc.c
+++ b/contrib/ntsc-cc.c
@@ -32,7 +32,6 @@
 #include <fcntl.h>
 #include <errno.h>
 #include <locale.h>
-#include <sys/ioctl.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/time.h>
@@ -45,6 +44,7 @@
 #include "src/libzvbi.h"

 #ifdef ENABLE_V4L2
+#  include <sys/ioctl.h>
 #  include <asm/types.h>
 #  include "src/videodev2k.h"
 #endif
--
2.40.0

